Sources/BraintreePaymentFlow/BTPaymentFlowDriver.m

```diff
@@ -74,6 +74,7 @@ - (void)performSwitchRequest:(NSURL *)appSwitchURL {
 
 - (void)informDelegatePresentingViewControllerRequestPresent:(NSURL *)appSwitchURL {
     if ([self.viewControllerPresentingDelegate respondsToSelector:@selector(paymentDriver:requestsPresentationOfViewController:)]) {
+        // Why not ASWebAuthenticationSession here?
```

Comment:
After conversation in stand-up, we decided we will not make this change to BTPaymentFlowDriver at the moment. It requires the user to agree to an alert view, which has the `braintreegateway.com` URL. We tabled the work for it but left a working branch up at `aswebauth-payment-flow-driver`.

![Uploading Screen Shot 2021-02-03 at 1.03.06 PM.png…]()